docs/source/using/addons.adoc

```diff
@@ -161,6 +161,23 @@ $ minishift addons enable anyuid
 $ minishift addons disable anyuid
 ----
 
+[[apply-addons]]
+== Applying add-ons
+
+Add-ons can be explicitly executed with the link:../command-ref/minishift_addons_apply{outfilesuffix}[`minishift addons apply`] command.
```

Comment:
@Preeticp I disagree  with the use of `also` as this is first sentence of the section.